addons/switchunits/functions/fnc_markAiOnMap.sqf

```diff
@@ -10,28 +10,24 @@
  * None
  *
  * Example:
- * [[west, east]] call FUNC(markAiOnMap)
+ * [[west, east]] call ace_switchunits_fnc_markAiOnMap
  *
  * Public: No
  */
-
 #include "script_component.hpp"
 
-private "_sidesToShow";
-_sidesToShow = _this select 0;
+params ["_sidesToShow"];
 
 GVAR(AllMarkerNames) = [];
 
-DFUNC(pfhMarkAiOnMap) = {
-    private ["_args", "_sides"];
-    _args = _this select 0;
-    _sides = _args select 0;
-
+_fnc_pfhMarkAiOnMap = {
```

Comment:
Also a missing private. Same thing, as with the other, might be better to put the code straight into the PFH without an assignment.